src/main/java/pl/isa/biblioteka/Menu.java

```diff
@@ -1,46 +1,59 @@
 package pl.isa.biblioteka;
 
-import java.util.List;
 import java.util.Scanner;
 
-import static pl.isa.biblioteka.FolderBooks.readBooks;
-
 public class Menu {
-    public void ShowMenu() {
+
+    public void selectUser() {
+
+        System.out.println("\nWitamy w naszej bibliotece");
+        System.out.println("Kim jesteś ?\nWybierz odpowiednią opcję w menu");
+        System.out.println("1 - Bibliotekarz");
+        System.out.println("2 - Użytkownik");
+
+        Scanner scanner = new Scanner(System.in);
+        try {
+            int userChoose1 = scanner.nextInt();
```

Comment:
userChoose1 - po co to 1?

Reply:
Coś tam testowałem kiedy nie chciało działać jak oczekiwałem i zmieniłem nazwę tej zmiennej po naprawieniu oczywiście nie wróciłem do original.
Już poprawiłem
Dodałem też zmiany od Kingi i poprawiłem bo jakoś nadpisało mi znów jedną sekcję menu.